addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2613,6 +2613,7 @@ we-select.o_grid we-toggler {
         stroke: $o-we-bg-darkest;
         stroke-opacity: .2;
         stroke-width: 1px;
+        filter: drop-shadow(-1px -1px 0px rgba(255, 255, 255, 0.3));
```

Comment:
With this filter, there is more contrast when the background is dark, making the grid easier to see. Even if the rectangles are not that "more visible", the trick of this filter is that it shifts the shadow left and up by 1px, making the border look like a double line (a dark one and a white one) so it will always be visible depending on the background :slightly_smiling_face:  

With the filter: 
![image](https://github.com/odoo/odoo/assets/102796037/66d0e755-d878-4e2f-b8c2-d5c7b6f7e56c)

Without the filter:
![image](https://github.com/odoo/odoo/assets/102796037/fc5d3ca0-dbcf-4eed-ad1d-d76a5f5f4122)